app/Models/Mship/Account.php

```diff
@@ -257,6 +257,11 @@ public static function findOrRetrieve($accountId)
 
             $account = self::find($accountId);
 
+            if (!$account) {
+                // User doesn't exist at VATSIM.NET
+                return false;
```

Comment:
In accordance with the rest of the flows within this function, would it not be more germane to throw an exception here instead?

Reply:
Yeah fair point, I had been thinking about doing that